docs/authorization-and-authentication/docs/012-cr-idppresets.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Identity Provider Presets
+type: Custom Resource
+---
+
+The `idppresets.authentication.kyma-project.io` Custom Resource Definition (CRD) is a detailed description of the kind of data and the format that represents presets of Identity Provider configuration used in securing API through console UI. Presets are a convenient way to configure authentication section in API Custom Resource.
```

Comment:
Regarding bolding the authentication section - it is a field of the API CR. I think that it is up to you - technical writers - to decide if references to fields of other CRs should be bolded.

Regarding "Presets" vs "presets" - I would rather use not capitalized word. Also, I used "Presets" capitalized only when referring to the resource (IDPPresets/IDP Presets) and "presets" when it was about preset as a predefined setting which is a result of creating IDPPreset CR and which can be seen for example in the dropdown menu. Maybe I should use "IDPPresets" instead of "IDP Presets" rather than using "Presets" instead of "presets"? What do you think?

Reply:
I like your suggestion to use "IDPPresets" instead of "IDP Presets" and not to capitalize "presets" themselves :)

Reply:
As for the **authentication**, it should be put in bold :)